tests/js/client/load-balancing/load-balancing-transactions-noauth-cluster.js

```diff
@@ -250,6 +254,69 @@ function TransactionsSuite () {
       }
     },
 
+    testUseTransaction: function () {
+      let trx;
+      try {
+        const obj = { collections: { read: cn } };
+        let result = sendRequest('POST', "/_api/transaction/begin", obj, true);
+        assertEqual(result.status, 201);
+        assertFalse(result.body.result.id === undefined);
```

Comment:
Simplification for next time (note: also requires adding `assertNotUndefined` to the list of globals in the prelude)
```suggestion
        assertNotUndefined(result.body.result.id);
```